packages/core/test/unit/utils/file-utils.spec.ts

```diff
@@ -22,31 +22,46 @@ describe('fileUtils', () => {
     });
   });
 
-  describe('findNodeModules', () => {
-    it('should return node_modules located in `basePath`', async () => {
-      statStub.resolves();
+  describe('findNodeModulesList', () => {
+    it('should return node_modules array located in `basePath`', async () => {
       const basePath = path.resolve('a', 'b', 'c');
       const expectedNodeModules = path.join(basePath, 'node_modules');
-      const actual = await fileUtils.findNodeModules(basePath);
-      expect(actual).eq(expectedNodeModules);
+      readdirStub.resolves([expectedNodeModules]);
+      const actual = await fileUtils.findNodeModulesList(basePath);
+      expect(actual.nodeModulesList[0]).eq(expectedNodeModules);
     });
 
-    it("should return node_modules located in parent directory of `basePath` if it didn't exist in base path", async () => {
+    it("should return node_modules array located in parent directory of `basePath` if it didn't exist in base path", async () => {
       const basePath = path.resolve('a', 'b', 'c');
       const expectedNodeModules = path.resolve('a', 'node_modules');
-      statStub
-        .throws() // default
-        .withArgs(expectedNodeModules)
-        .resolves();
-      const actual = await fileUtils.findNodeModules(basePath);
-      expect(actual).eq(expectedNodeModules);
+      readdirStub.withArgs(path.dirname(path.resolve(basePath))).resolves([expectedNodeModules]);
+      const actual = await fileUtils.findNodeModulesList(basePath);
+      expect(actual.nodeModulesList[0]).eq(expectedNodeModules);
     });
 
-    it('should return null if no node_modules exist in basePath or parent directories', async () => {
+    it('should return node_modules array in subDirectory of `basePath`', async () => {
+      const basePath = path.resolve('a');
+      const expectedNodeModulesList = [path.resolve('a', 'b', 'node_modules'), path.resolve('a', 'b', 'c', 'node_modules')];
+      readdirStub.withArgs(path.resolve(basePath)).resolves([path.resolve(basePath, 'b')]);
```

Comment:
`readdir` won't ever return a resolved path. It is probably better to let it resolve `'b'` instead.

Reply:
Yes, sorry for this mistake, I fix this now